src/state/ViewerState.tsx

```diff
@@ -78,6 +85,19 @@ class ViewerState {
     }
     setIsLoggedIn(newState: boolean) {
         this.isLoggedIn = newState
+        if (this.isLoggedIn){
+            // Cache user_uuid until logout
+            const userName = localStorage.getItem('CognitoIdentityServiceProvider.6jlm2jeibh9aqb0dg34q2uf8pu.LastAuthUser');
+            const storedDataString = localStorage.getItem('CognitoIdentityServiceProvider.6jlm2jeibh9aqb0dg34q2uf8pu.'+userName+'.userData');
```

Comment:
Same here with the CognitoServiceProvider, are those ID ok here? I think it is but just want to make sure it is not sensible information.